code/modules/power/singularity/emitter.dm

```diff
@@ -74,9 +74,9 @@
 	rotate()
 
 /obj/machinery/power/emitter/Destroy()
-	msg_admin_attack("Emitter deleted at ([x],[y],[z] - [ADMIN_JMP(src)])", ATKLOG_FEW)
+	msg_admin_attack("Emitter deleted at ([x],[y],[z] - [ADMIN_JMP(src)]) [usr ? "Broken by [key_name_admin(usr)]" : ""]", ATKLOG_FEW)
```

Comment:
Can this instead be logged using `add_attack_logs(usr, target, "broke text", ATKLOG_FEW)` if the usr is not null?